server/routes/error/locale-bad-request.json

```diff
@@ -0,0 +1,15 @@
+{
+  "en": {
+    "page": {
+      "title": "There was an issue with your request - {{ applicationName }}"
+    },
+    "mainHeading": {
+      "title": "There was an issue with your request"
+    },
+    "suggestions": [
+      "Try performing your request again",
+      "If the request still fails, try again later."
+    ],
+    "resolution": "<a href='{{ oasysReturnUrl }}' class='govuk-link govuk-link--no-visited-state'>Go to the OASys homepage</a>"
```

Comment:
I'm sure you complained about links in the locale in another PR. 😉

Reply:
I did.... I'm my own worst enemy 🙄 It's a bit of a hacky solution, there is a slightly more long-winded way of handling it, but I ended up kind of torn between thinking is this acceptable or not. It's like, if it was a CMD, it would store this in the text block, the same with <strong> tags, <italics> etc. I'm still very torn on if this is the right approach.